Makefile

```diff
@@ -442,8 +442,7 @@ ifneq '' '$(findstring mingw,$(shell $(CC) -dumpmachine))'
 	MK_CFLAGS   += -Xassembler -muse-unaligned-vector-move
 	MK_CXXFLAGS += -Xassembler -muse-unaligned-vector-move
 
-	# Target Windows 8 for PrefetchVirtualMemory
-	MK_CPPFLAGS += -D_WIN32_WINNT=0x602
+	MK_CPPFLAGS += -D_WIN32_WINNT=_WIN32_WINNT_WIN7 -DWINVER=_WIN32_WINNT_WIN7
```

Comment:
> I'm not sure if this has any effects on other versions of windows

According to <https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt>: "The preprocessor macros WINVER and _WIN32_WINNT specify the minimum operating system version your code supports".

This will prevent windows sdk header file declare functions only available on higher windows versions. So as long as the project can compile, it should have no effects on other windows versions.